Manifests/ManagedPreferencesApplications/com.papercut.printdeploy.plist

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>pfm_description</key>
+	<string>PaperCut Print Deploy Settings</string>
+	<key>pfm_documentation_url</key>
+	<string>https://www.papercut.com/help/manuals/print-deploy/roll-out-the-client/install-arguments/#available-parameters-and-filename-configurations</string>
+	<key>pfm_domain</key>
+	<string>com.papercut.printdeploy.client</string>
+	<key>pfm_format_version</key>
+	<integer>1</integer>
+	<key>pfm_last_modified</key>
+	<date>2023-03-07T23:00:00Z</date>
+	<key>pfm_platforms</key>
+	<array>
+		<string>macOS</string>
```

Comment:
Are the bundle ID and preference names the same for your iOS client? If so it might be useful to add iOS here as well (for plist export).

Reply:
Based on the documentation, these settings wouldn't apply to iOS devices. iOS Devices don't support the Print Deploy Client, just Mobility Print. Whereas macOS devices support both Print Deploy and Mobility Print.